lib/tabry/shells/fish/wrapper.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require "shellwords"
+require "yaml"
+require_relative "../../util"
+require_relative "../../runner"
+require_relative "../../shell_tokenizer"
+
+# Fish-specific entrypoint
+module Tabry
+  module Fish
+    module Wrapper
+      DESCRIPTION_MAX_LENGTH = 50
+
+      def self.run(cmd_line, comp_point, config: nil)
+        cmd_name, args, last_arg = Tabry::ShellTokenizer.split_with_comppoint(cmd_line, comp_point)
+        opts = Tabry::Runner.new(config: config || cmd_name).options(args, last_arg, {descriptions: true})
+
+        if Tabry::Util.debug?
+          require "json"
```

Comment:
This require doesn't seem to be used

Reply:
Ah okay, I just copied from the bash counterpart: https://github.com/evanbattaglia/tabry/blob/master/lib/tabry/shells/bash/wrapper.rb#L18